charts/rancher-logging/0.2.5/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+description: Rancher logging helm chart to support logging function in rancher
+icon: https://raw.githubusercontent.com/fluent/fluentd-docs/master/public/logo/Fluentd_square.png
+name: rancher-logging
+version: 0.2.5
```

Comment:
This is also not in "new charts" issue: https://github.com/rancher/rancher/issues/33528
I believe we talked about this, but can someone track down.